docs/tutorials/02_neural_network_classifier_and_regressor.ipynb

```diff
@@ -95,14 +95,11 @@
     "y01 = 1*(np.sum(X, axis=1) >= 0)  # in { 0,  1}\n",
     "y = 2*y01-1                       # in {-1, +1}\n",
     "y_one_hot = np.zeros((num_samples, 2))\n",
-    "for i in range(num_samples):\n",
-    "    y_one_hot[i, y01[i]] = 1\n",
-    "\n",
-    "for x, y_target in zip(X, y):\n",
-    "    if y_target == 1:\n",
-    "        plt.plot(x[0], x[1], 'bo')\n",
-    "    else:\n",
-    "        plt.plot(x[0], x[1], 'go')\n",
+    "y_one_hot[y01 == 1] = [0, 1]\n",
+    "y_one_hot[y01 == 0] = [1, 0]\n",
+    "\n",
+    "plt.plot(X[y == 1, 0], X[y == 1, 1], 'bo')\n",
+    "plt.plot(X[y != 1, 0], X[y != 1, 1], 'go')\n",
```

Comment:
Same issue as in the torch connector tutorial. I'd say too pythonic and may require additional explanation to a user.